server/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import express from "express";
+import bodyParser from "body-parser";
+import compression from "compression";
+import cors from "cors";
+import mongoose from "mongoose";
+import router from "./router";
+import "dotenv/config";
+
+const app = express();
+
+app.use(
+  cors({
+    credentials: true,
+  })
+);
+
+app.use(compression());
+app.use(bodyParser.json());
+
+mongoose.Promise = Promise;
+
+mongoose
+  .connect(process.env.MONGO_URL)
```

Comment:
you can pass the connection string without the query parameters ( ["mongodb+srv://emanueldrusu:[Mongowebgurus22@wgclusterlearn.gr0pdlo.mongodb.net](mailto:Mongowebgurus22@wgclusterlearn.gr0pdlo.mongodb.net)](mongodb+srv://biroaron:123@cluster0.y7nbxkk.mongodb.net/)" ) and add those here in the options :
.connect(connectionString, { retryWrites: true, w: 'majority', dbName: '...' })